core/static/css/wf_website.css

```
@@ -161,3 +161,9 @@ p {
 .navbar-dark .navbar-nav .nav-link {
   color: #ffe9e5;
 }
+
```

Let's add a reminder comment here.

```suggestion
/* Make sure embedded videos take up full with 
while preserving the typical aspect ratio */
```